main/src/ca/uwaterloo/flix/language/phase/jvm/GenChannelClasses.scala

```diff
@@ -158,6 +161,25 @@ object GenChannelClasses {
     isEmpty.visitEnd()
   }
 
+  def genNonEmpty(classType: JvmType.Reference, channelType: JvmType, visitor: ClassWriter)(implicit root: Root, flix: Flix): Unit = {
+    val nonEmpty = visitor.visitMethod(ACC_PUBLIC, "nonEmpty", AsmOps.getMethodDescriptor(Nil, JvmType.PrimBool), null, null)
+    val l0 = new Label()
+    val l1 = new Label()
+    nonEmpty.visitCode()
+    nonEmpty.visitVarInsn(ALOAD, 0)
+    nonEmpty.visitFieldInsn(GETFIELD, classType.name.toInternalName, "queue", JvmType.LinkedList.toDescriptor)
+    nonEmpty.visitMethodInsn(INVOKEINTERFACE, "java/util/LinkedList", "isEmpty", AsmOps.getMethodDescriptor(Nil, JvmType.PrimBool), true)
```

Comment:
Use `JvmType.LinkedList.name.toInternalName` for making `"java/util/LinkedList"`